packages/blocks/src/__internal__/rich-text/shortcuts.ts

```diff
@@ -286,6 +286,7 @@ export class Shortcuts {
         };
         page.deleteBlock(model);
         page.addBlock(blockProps, parent, index);
+        page.captureSync();
```

Comment:
How it works

Reply:
It's not related to hotkey manager, Just unify behavior of after creating a block, use page.captureSync() :)

Reply:
But in most cases, we call `page.captureSync()` before block operation

Reply:
`page.captureSync()` is now before block operation